sonic_package_manager/manifest.py

```diff
@@ -213,6 +213,8 @@ def unmarshal(self, value):
         ManifestRoot('container', [
             ManifestField('privileged', DefaultMarshaller(bool), False),
             ManifestArray('volumes', DefaultMarshaller(str)),
+            ManifestField('network', DefaultMarshaller(str), ''),
```

Comment:
I assume the command ```sonic-package-manager show package manifest``` will display empty string ```""``` as a value of ```network``` field. Do you want the default value to be ```'host'``` rather then empty string?